src/modules/musig/tests_impl.h

```diff
@@ -1328,19 +1329,91 @@ void musig_test_vectors_sign(void) {
             0x19, 0x5C, 0x1D, 0x4B, 0x52, 0xE6, 0x3E, 0xCD,
             0x7B, 0xC5, 0x99, 0x16, 0x44, 0xE4, 0x4D, 0xDD,
         };
-        const unsigned char tweak[32] = {
+        const unsigned char tweak[1][32] = {{
             0xE8, 0xF7, 0x91, 0xFF, 0x92, 0x25, 0xA2, 0xAF,
             0x01, 0x02, 0xAF, 0xFF, 0x4A, 0x9A, 0x72, 0x3D,
             0x96, 0x12, 0xA6, 0x82, 0xA2, 0x5E, 0xBE, 0x79,
             0x80, 0x2B, 0x26, 0x3C, 0xDF, 0xCD, 0x83, 0xBB,
-        };
-        musig_test_vectors_sign_helper(&keyagg_cache, &fin_nonce_parity, sig, secnonce, agg_pubnonce, sk, msg, tweak, 0, NULL, pk, 2);
+        }};
+        int is_xonly_t[1] = { 0 };
+        musig_test_vectors_sign_helper(&keyagg_cache, &fin_nonce_parity, sig, secnonce, agg_pubnonce, sk, msg, tweak, is_xonly_t, 1, NULL, pk, 2);
```

Comment:
nit:
```suggestion
        int is_xonly_t[1] = { 0 };
        CHECK(sizeof(is_xonly_t) == sizeof(tweak)/sizeof(tweak[0]));
        musig_test_vectors_sign_helper(&keyagg_cache, &fin_nonce_parity, sig, secnonce, agg_pubnonce, sk, msg, tweak, is_xonly_t, sizeof(is_xonly_t), NULL, pk, 2);
```
this makes the code a little more robust against copy-n-paste mistakes.

same below.